src/schedule/zephyr_ll.c

```diff
@@ -176,6 +176,9 @@ static void zephyr_ll_run(void *data)
 	struct list_item *list, *tmp, task_head = LIST_INIT(task_head);
 	uint32_t flags;
 
+	notifier_event(sch, NOTIFIER_ID_LL_PRE_RUN,
+		       NOTIFIER_TARGET_CORE_LOCAL, NULL, 0);
```

Comment:
this notifier is only directed to the same core as the one this LL scheduler is running on. So the notification handler will be called inline. Is this really what we want?

Reply:
as for now - yes. 
Later we need to unify all DP schedulers for all cores - as one DP may affect other regardless of a core they are located on.

Reply:
@marcinszkudlinski but that also means then that those notification handlers on this core will run in the LL scheduler context, thus delaying LL tasks

Reply:
I think it is indeed wrong that the context and the priority of the callback depend on the core it happens to run on.